tools/scoring/score/parse_config.py

```diff
@@ -220,3 +236,49 @@ def aggregate_results(
 
         results[dimension.__name__] = invoked
     return results
+
+  # TODO: standardize signatures; make dimensions into const; test
+  def execute(self) -> Dict[DimensionName, str]:
+    """
+      Wrapper for all functionality herein.
+
+      Returns:
+        Dictionary containing human-readable
+        represenation of every scored dimension.
+    """
+    deserialized_files_appended = self.append_types(
+        universe=self.universe, deserialized_files=self.deserialized_files)
+
+    matches = self.match_reporting_entities(
+        proposed_entities=deserialized_files_appended[PROPOSED],  # pylint: disable=unsubscriptable-object
+        solution_entities=deserialized_files_appended[SOLUTION])  # pylint: disable=unsubscriptable-object
+
+    translations = self.retrieve_reporting_translations(
+        matches=matches,
+        proposed_entities=deserialized_files_appended[PROPOSED],  # pylint: disable=unsubscriptable-object
+        solution_entities=deserialized_files_appended[SOLUTION])  # pylint: disable=unsubscriptable-object
+
+    dimensions = {
+        SIMPLE: [
+            raw_field_selection.RawFieldSelection,
+            standard_field_naming.StandardFieldNaming,
+            state_mapping.StateMapping, unit_mapping.UnitMapping
+        ],
+        COMPLEX: [
```

Comment:
what is the difference between simple and complex? in the sense, if I want to add a new dimension. How do I know where to add it?

Reply:
It's noted in `constants.py` and in `dimension.py`. Should I explain here too?